src/include/parser/copy.h

```diff
@@ -33,5 +34,21 @@ class Copy : public Statement {
     std::unordered_map<std::string, std::unique_ptr<ParsedExpression>> parsingOptions;
 };
 
+class CopyTo : public Statement {
+public:
+    explicit CopyTo(std::string filePath, std::unique_ptr<RegularQuery> regularQuery)
+        : Statement{common::StatementType::COPY_TO},
+          regularQuery{std::move(regularQuery)}, filePath{std::move(filePath)} {}
+
+    inline std::string getFilePath() const { return filePath; }
+    inline std::unique_ptr<RegularQuery> getRegularQuery() { return std::move(regularQuery); }
+
+private:
+    std::string filePath;
+    common::CopyDescription::FileType fileType;
```

Comment:
I guess you only know file type after binding. And it seems this field is not being used. So remove it.